.ci/run-tests.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+
+# Function to display help
+show_help() {
+    echo "Usage: $0 [options]"
+    echo "Options:"
+    echo "  --force-recreate    Recreate and build containers before running tests"
+    echo "  --help              Show this help message"
+}
```

Comment:
Nit, can also do:

```bash
show_help() {
cat<<FIN
Usage: $0 [options]
Options:
   --force-recreate Recreate and build containers before running tests
   --help Show this help message
FIN
}
```

Also `$0` will show the whole path, not the name of the program. For the name of the program, it's better `$(basename "$0")`